docs_versioned_docs/version-ros2humble/robots/accessories/manipulators/kinova_gen3_lite.mdx

```diff
@@ -210,78 +154,6 @@ If you cannot ping the computer or arm, try rebooting and replugging the Kinoca
 
 :::
 
-### Launching the ROS Driver {#kinova_gen3_lite_launch}
```

Comment:
There is a reference to #kinova_gen3_lite_launch that now no longer resolves correctly. The reference should be fixed or removed.